conceptual/Npgsql/release-notes/8.0.md

```diff
@@ -64,6 +64,27 @@ Version 8.0 contains many other smaller features and bug fixes, [see the 8.0.0 m
 
 ## Breaking changes
 
+### Synchronous command batching can cause threadpool starvation
+
+To allow Npgsql to write command batches containing large data payloads (generally parameter data) synchronously, without running into PostgreSQL limitations, it runs any commands after the first asynchronously. This introduces sync over async and can cause threadpool starvation at higher loads. In Npgsql versions prior to 8.0 a workaround was in place that would mitigate the effects of this sync over async, allowing most higher loads to run succesfully. This workaround was removed to make room for the far reaching NativeAOT refactor.
+
+Affected APIs are the synchronous command execution methods: ExecuteReader, ExecuteNonQuery, ExecuteScalar in combination with a command batch (i.e. NpgsqlBatch or semicolon ';' concatenated commands).
+
+If you are affected by this you can consider some of the following options:
+
+* Use [ThreadPool.SetMinThreads()](https://learn.microsoft.com/en-us/dotnet/api/system.threading.threadpool.setminthreads?view=net-8.0) to expand the minimum pool of threads available for blocking.
+* Refactor code using command batches to use asynchronous apis.
+
+If these options are not suitable for your project you can consider opening an issue on [https://github.com/npgsql/npgsql](https://github.com/npgsql/npgsql). This helps us prioritize the effort to bring back some mitigation in a patch version.
+
+To clarify, Npgsql 8.0 has not dropped support for synchronous command batches completely; the feature is entirely supported for use at low load or for local data exploration.
```

Comment:
Is that maybe a bit too harsh? Given the workaround (e.g. SetMinThreads()) it's possible to do this even at higher load, it's just up to the user to preconfigure things etc?